packages/@ember/-internals/extension-support/lib/container_debug_adapter.ts

```diff
@@ -1,13 +1,9 @@
 import { classify, dasherize } from '@ember/string';
-import {
-  A as emberA,
-  typeOf,
-  Namespace,
-  Object as EmberObject,
-  NativeArray,
-} from '@ember/-internals/runtime';
-import { getOwner, Owner } from '@ember/-internals/owner';
-import { Resolver } from '@ember/-internals/container/lib/registry';
+import type { NativeArray } from '@ember/-internals/runtime';
+import { A as emberA, typeOf, Namespace, Object as EmberObject } from '@ember/-internals/runtime';
```

Comment:
Same issue here. This should actually be:

```suggestion
import {
  A as emberA,
  typeOf,
  Namespace,
  Object as EmberObject,
  type NativeArray,
} from '@ember/-internals/runtime';
```

🤔 I think I’m probably fine with taking it even so, but it’s quite annoying.

Do you mind testing that the lint rule minimally doesn't *reject* this form, even if it doesn't auto-fix *to* it?

Reply:
I does reject it and auto fix doesn't fix it.

<img width="277" alt="Screen Shot 2022-05-10 at 8 13 38 AM" src="https://user-images.githubusercontent.com/7374640/167636648-01291760-2150-40e1-8a2f-317f93d0505d.png">
.